docs/versioned_docs/version-2.11.0/user-authentication/sso/ldap.md

```diff
@@ -0,0 +1,67 @@
+---
+id: ldap
+title: LDAP
+---
+
+To set up LDAP as Single Sign-On (SSO) for ToolJet, follow these steps:
+
+1. Access the ToolJet dashboard and click on the ⚙️ icon to open **Workspace Settings** from the left sidebar.
+
+  <div style={{textAlign: 'center'}}>
+
+  <img className="screenshot-full" src="/img/sso/ldap/settings.png" alt="SSO :LDAP" width="500"/>
+
+  </div>
+
+2. In the Workspace Settings, navigate to the **SSO** section and choose **LDAP**. By default, LDAP login will be **disabled**.
+
+  <div style={{textAlign: 'center'}}>
+
+  <img className="screenshot-full" src="/img/sso/ldap/disabled.png" alt="SSO :LDAP"/>
+
+  </div>
+
+3. To **enable** LDAP, toggle the switch. Then, add the configuration:
+
+   - **Name**: Enter the name of the SSO.
+   - **Hostname**: Provide the hostname or IP address of your LDAP server.
+   - **Port**: Enter the Port number of LDAP server.
+   - **Base DN**: Enter the base distinguished name.
+   - **SSL**: Toggle this option to enable the SSL. After enabling you can select the type of SSL: **None** or **Certificates**. If you choose Certificates, you'll need to provide the **Client Key**, **Client Certificate**, and **Server Certificate**.
+
+  <div style={{textAlign: 'center'}}>
+
+  <img className="screenshot-full" src="/img/sso/ldap/fields.png" alt="SSO :LDAP"/>
+
+  </div>
+
+4. After making the necessary configurations, click the Save Changes button located at the bottom.
+
+
+5. Next, proceed to the **[General Settings](/docs/user-authentication/general-settings)** and copy the **Login URL** provided. In the General Settings, you have the option to **Enable Signups** if you wish to automatically add new users when their accounts are not found in the SSO. If you prefer to disable this option, users will receive an error if their accounts are not found.
```

Comment:
```suggestion
5. Next, proceed to the **[General Settings](/docs/user-authentication/general-settings)** and copy the **Login URL** provided. Furthermore, you have the flexibility to choose whether to turn on 'Enable Signups,' allowing users to signup without an invite. Through SSO authentication, we check if the user already exists; if so, they can sign in seamlessly. Otherwise, an error will be displayed. Conversely, with this option disabled, only invited users can log in, provided SSO authentication is successful.
```